### DISCUSSION TITLE
Strip ANSI color codes from the source command output but not from the preview command

### DISCUSSION THREAD

**Discussion by andreypopp** [junegunn/fzf#2653] [Q&A] [answered]
Is it possible to instruct `fzf` to strip all colors from the source command output but keep colors coming from the preview command?

I've found out that setting `--color=bw` makes it strip color codes but it works for both list items and the preview.

**Comment by vovcacik** [comment, marked as answer]
None that I know of. Disable the colored output, or put some stripper in between, e.g. `git status | stripansi | fzf`. [Program that passes STDIN to STDOUT with color codes stripped?](https://unix.stackexchange.com/questions/4527/program-that-passes-stdin-to-stdout-with-color-codes-stripped)
